src/appshell/appshell.cpp

```diff
@@ -70,6 +70,16 @@ int AppShell::run(int argc, char** argv)
         appName  = "MuseScore4";
     }
 
+#ifdef Q_OS_WIN
+    // NOTE: There are some problems with rendering the application window on some integrated graphics processors
+    //       see https://github.com/musescore/MuseScore/issues/8270
+    QCoreApplication::setAttribute(Qt::AA_UseOpenGLES);
+
+    if (!qEnvironmentVariableIsSet("QT_OPENGL_BUGLIST")) {
+        qputenv("QT_OPENGL_BUGLIST", ":/resources/win_opengl_buglist.json");
```

Comment:
BTW it works fine on my system without this, just setting AA_UseOpenGLES is sufficient.

Reply:
yes, but there may be problems with these processors and it is very difficult to understand what the problem is.
insure against them

Reply:
> BTW it works fine on my system without this, just setting AA_UseOpenGLES is sufficient.

See ed4949c as to why it was needed in 3.x, there's no reason to assume it isn't needed in master, as far as I can tell